content/docs/get-started/index.md

```diff
@@ -279,238 +246,258 @@ Alternatively, you could save the dataset itself
-local FastAPI model server, simply run:
+To launch a FastAPI model server locally, simply run:
 
 ```cli
 $ mlem serve fastapi --model models/rf
-⏳️ Loading model from models/rf.mlem
-Starting fastapi server...
-🖇️  Adding route for /predict
-🖇️  Adding route for /predict_proba
-🖇️  Adding route for /sklearn_predict
-🖇️  Adding route for /sklearn_predict_proba
-Checkout openapi docs at <http://0.0.0.0:8080/docs>
-INFO:     Started server process [16696]
-INFO:     Waiting for application startup.
-INFO:     Application startup complete.
-INFO:     Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)
 ```
 
-The server is now running and listening on `http://0.0.0.0:8080` for HTTP
-requests.
+🤩 The server is now running, **serving your model**, and listening on
+`http://0.0.0.0:8080` for HTTP requests.
 
-Servers automatically create endpoints from model methods using the
-`sample_data` argument provided to [`mlem.api.save`](/doc/api-reference/save).
-
-<admon type="tip">
-
-Servers automatically create endpoints from model methods using the
-`sample_data` argument provided to [`mlem.api.save`](/doc/api-reference/save).
-
-</admon>
-
-#### Making requests
+### Making requests
 
 While the model server is running, you can use your browser to open the Swagger
-UI (OpenAPI) at [http://localhost:8080/docs](http://localhost:8080/docs) and
-check out OpenAPI spec and query examples.
+UI at [http://localhost:8080/docs](http://localhost:8080/docs) and check out
+OpenAPI spec and query examples.
 
 <admon type="info">
 
-MLEM can also generate client-side code to query/infer the model server. Learn
-more about this in
-[Serving User Guide](/doc/user-guide/serving#making-requests).
+MLEM can also generate a client code object to query/infer the model server.
 
 </admon>
 
-## Deploying models to production
+Check out the [serving guide](/doc/user-guide/serving) to learn more.
+
+## Deploying models in production
 
-Now, let's take model serving a step further and use production worthy
-deployment technologies. MLEM lets you easily package and deploy your models
-using a variety of platforms like [Docker](/doc/user-guide/deploying/docker),
+💪 Now, let's take model serving a step further.
+
+MLEM lets you easily package and deploy your models to a variety of platforms
+like [Docker](/doc/user-guide/deploying/docker),
 [Heroku](/doc/user-guide/deploying/heroku),
 [Sagemaker](/doc/user-guide/deploying/sagemaker) and
-[Kubernetes](/doc/user-guide/deploying/kubernetes), so you don't have to deal
-with the DevOps and implementation details of deployment yourself. See the full
-list in the [Deploying User Guide](/doc/user-guide/deploying).
+[Kubernetes](/doc/user-guide/deploying/kubernetes). This way, you don't have to
+deal with the DevOps and implementation details of the servers and deployments.
+
+<admon type="warn">
+
+**Security Notice:** Be careful when exposing resources / servers on a public
```

Comment:
I like this, sounds serious 🕶️

Reply:
We also don't want people "blaming" mlem for making deployment easy but exposing their infrastructure to unnecessary risks - makes it feel like a toy and not ready for production usage.
But I'm not comfortable with not having a use case showing how to add authentication to your server to link to yet!